Makefile

```diff
@@ -1,6 +1,7 @@
 DEST := /usr/local/bin/vapor
 
 build:
+        swift build -c release
```

Comment:
Is this needed? Seems a bit redundant

Reply:
I found that if I just built it without the release flag, it defaulted to debug.

Which ultimately changed the output folder.

Reply:
Right, sorry I mean why are we doing a build here at all? The `build.swift` script is the one doing the build

Reply:
Ah! Duh. Apologies, should be fixed up now.